Scripts/SwiftLint.sh

```diff
@@ -26,8 +26,19 @@ if [ -z "$selected_path" ]; then
 	exit 1
 fi
 
+# copy the generated output to a temp dir and strip the "swiftlint:disable:all"
 if [ "$key" = "templates_generated" ]; then
-	find "${PROJECT_DIR}/${selected_path}" -name "*.swift" -exec ${PROJECT_DIR}/Scripts/swiftlint_no_disable_all.sh {} \;
+	temp_dir=`mktemp -d -t SwiftGen`
+
+	for f in `find "${PROJECT_DIR}/${selected_path}" -name '*.swift'`; do
+		temp_file="${temp_dir}${f#*"$PROJECT_DIR"}"
+		mkdir -p $(dirname "$temp_file")
+		cat "$f" | sed s@"swiftlint:disable all"@" --"@ > "$temp_file"
+	done
+
+	files_path="$temp_dir"
 else
-	"$SWIFTLINT" lint --strict --config "$CONFIG" --path "${PROJECT_DIR}/${selected_path}"
+	files_path="${PROJECT_DIR}/${selected_path}"
 fi
+
+"$SWIFTLINT" lint --strict --config "$CONFIG" --path "${files_path}"
```

Comment:
Might be worth explicitly/manually remove the temp dir before exiting? (Or does mkdir do that automatically on process exit?)

Either by wrapping that in another `if` here at the end of the script, or by using the `trap` shell command after the mkdir to program a deferred removal.